src/tsp_worker.h

```diff
@@ -10,33 +10,40 @@
 #include <utility>
 #include <vector>
 
-struct TSPWorker final : Nan::AsyncWorker {
-  using Base = Nan::AsyncWorker;
+struct TSPWorker final {
```

Comment:
Why did you remove this and no longer initialize the base class below? The base class already stores the callback, there should be no need to store it here.